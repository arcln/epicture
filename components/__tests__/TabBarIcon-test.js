import {View} from 'react-native';
import React from 'react';
import TabBarIcon from '../TabBarIcon';
import renderer from 'react-test-renderer';

it('renders correctly focused', () => {
  const tree = renderer.create(
    <View style={{paddingTop: 10}}>
      <TabBarIcon
        focused={true}
        name={'ios-arrow-down'}
        size={42}
      />
    </View>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it('renders correctly not focused', () => {
  const tree = renderer.create(
    <View style={{paddingTop: 10}}>
      <TabBarIcon
        focused={false}
        name={'ios-arrow-down'}
        size={42}
      />
    </View>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
