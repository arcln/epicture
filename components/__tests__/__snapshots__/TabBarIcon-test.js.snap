// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly focused 1`] = `
<View
  style={
    Object {
      "paddingTop": 10,
    }
  }
>
  <Text />
</View>
`;

exports[`renders correctly not focused 1`] = `
<View
  style={
    Object {
      "paddingTop": 10,
    }
  }
>
  <Text />
</View>
`;
